internal/settings/settingsCmd.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
rename the file to `cmd` to avoid stuttering

Reply:
Done